mars-form/src/Component/form.js

```diff
@@ -0,0 +1,372 @@
+        this.setState({
+            [e.target.name]:e.target.value
+        })
+    }
+
+    render(){
+        console.log(this.state.whyMars)
+
+        
+        return(
+            <div>
+                <form onSubmit={this.handleSubmit}>
+                <h1>Mars Registration form</h1>
+                    <label>
+                    Please Enter The Information
+                    <br></br>
+                     First Name : <input placeholder = "insert first name" onChange = {this.handleFirstName} required/>
+                     <br/>
+                     Last Name : <input placeholder = "insert last name" onChange = {this.handleLastName} required/>
+                     <br/>
+                    </label>
+                    <label>
+                        Date of Birth: <input type="date" onChange = {this.handleDateOfBirth} required/>
+                    </label>
+                    <br/>
+                    <label>
+                        Country of Origin 
+                        <select value = {this.state.country} onChange = {this.handleSelect} key = "i" required>
+                            {this.populateSelect()}
+
+                        </select>     
+                        <br/>   
+                    </label>
+                    <label>
+                        Dietary Preferences
+                        <select value = {this.state.dietPref} onChange = {this.handleDiet} required>
+                            <option value="omnivore">Omnivore</option>
+                            <option value="vegetarian">Vegetarian</option>
+                            <option value="vegan">Vegan</option>
+                        </select>
+                    </label>
+                    <br/>
+                    <label>
+                        Why Do You Want To be A Mars Explorer?
+                        <br/>
+                        <textarea id="text" type="text"  onChange = {this.handleMars} required/>
+                    </label>
+                    <br/>
+                    <label>
+                        1.  Can you Breathe Underwater for longer then a minute?
+                        <br/>
+                            <label>
+                                <input
+                                     type="radio"
+                                     name="q1"
+                                     value="yes"
+                                     //checked= {"q1" === "yes"}
+                                     onChange={this.handleChange}
+
+                                />
+                                Yes
+                        </label>
+                        <label>
+                            <input
+                                type="radio"
+                                name="q1"
+                                value="no"
+                                //checked= {"q1" === "no"}
+                                onChange={this.handleChange}
```

Comment:
yours checks aren't working because you're comparing two strings. NOT a variable a string like it should. Don't commit out, fix*